api/v1alpha1/virtualmachine_types.go

```diff
@@ -392,6 +392,38 @@ type NetworkInterfaceStatus struct {
 	IpAddresses []string `json:"ipAddresses,omitempty"`
 }
 
+// NetworkInterfaceStatusEx defines the properties of network interface extended status attached to the VirtualMachineTemplate
+// as seen by OVF properties
+type NetworkInterfaceStatusEx struct {
```

Comment:
I don't think the `NetworkInterfaceStatusEx` should only contain one IPAddress. We should keep consistent with how net-operator and ncp does.